styles/prosilver/template/show_tag.html

```diff
@@ -108,7 +110,7 @@ <h2 class="forum-title">{RH_TOPICTAGS_SEARCH_HEADER}</h2>
 
 						<!-- EVENT topiclist_row_append -->
 					</div>
-				</dt>
```

Comment:
not true, see https://github.com/phpbb/phpbb/blob/3.2.x/phpBB/styles/prosilver/template/viewforum_body.html#L222